tests/functional/transaction_timeout.ts

```diff
@@ -99,8 +99,11 @@ describe('Transaction timout will reset the transaction id for the claim', async
     expected: true,
   })
 
-  await btcdClientB.generate(blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS + 1)
-  await btcdClientA.setNetworkActive(true)
+  await bitcoinCoreClientB.generate(blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS + 1)
+  const targetHeight = 102 + blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS
+  await waitForBlockchainSync(targetHeight, bitcoinCoreClientB)
```

Comment:
If this technique is going to be used in other tests in the future, it might be useful if `waitForBlockchainSync` took an array of clients and only returned once all of the clients were at the requested targetHeight.